src/index.tsx

```diff
@@ -118,8 +119,6 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
     if (config.throwOnFailedInitialization ?? true) {
       throw error;
     }
-  } finally {
```

Comment:
The `finally` call was invaliding the "throw" on line 120, causing this function to not actually throw when `throwOnFailedInitialization` was set to `true`

Reply:
Yuck! Good spot!